src/test/java/ibm/jceplus/junit/base/BaseTestRSAKeyInterop.java

```diff
@@ -536,7 +537,18 @@ public void testEncryptPlusDecryptJCE() {
 
             RSAPublicKey rsaPubPlus = (RSAPublicKey) rsaKeyPairPlus.getPublic();
             rsaKeyPairPlus.getPrivate();
-            Cipher cipherPlus = Cipher.getInstance("RSA/ECB/PKCS1Padding", providerName);
+            Cipher cipherPlus = null;
+            try {
+                cipherPlus = Cipher.getInstance("RSA/ECB/PKCS1Padding", providerName);
+            } catch (NoSuchAlgorithmException nsae) {
+                nsae.printStackTrace();
+                assertTrue("RSA is not instantiated in Cipher for " + providerName,
+                            (nsae instanceof NoSuchAlgorithmException && 
+                            nsae.getMessage().equals("No such algorithm: RSA/ECB/PKCS1Padding") &&
+                            providerName.equals("OpenJCEPlusFIPS"))
```

Comment:
We decided to follow the existing pattern of string comparison for now instead of calling a function such as `isFIPS()` we do acknowledge that this is prob a better long term approach to testing if the provider is fips or not.